docs/contributing.md

```diff
@@ -29,22 +29,25 @@ We typically will collect a group of issues or suggestions into a new topline fe
 
 Once a feature has been defined, typically it will be taken on by a product designer. Here, they will produce low fi mocks, get feedback from our users and community, and iterate.
 
-Once the main UX flows have been dialed in, there will be a hi-fidelity visual design. 
+Once the main UX flows have been dialed in, there will be a hi-fidelity visual design.
 
 Features that are ready for design are are tagged [Design Needed](https://github.com/metabase/metabase/labels/Design%2FNeeded). Once a feature has had a reasonably complete visual design it should be tagged [Help Wanted](https://github.com/metabase/metabase/labels/Help%20Wanted).
 
 ### D) Build the feature
 
 Once a feature is tagged [Help Wanted](https://github.com/metabase/metabase/labels/Help%20Wanted), it is considered ready to be built. A core team member (or you, awesomely helpful person that you are) can start working on it.
-Once one or more people have started to work on a feature, it should be marked [In Progress](https://github.com/metabase/metabase/labels/In%20Progress). Once there is a branch+some code, a pull request is opened, linked to the feature + any issues that were pulled together to inform the feature. 
+
+If you're building something that users will see in Metabase, please refer to the [Style Guide](https://localhost:3000/_internal) while running the development environment to learn how and when to use various Metabase UI elements.
+
+Once one or more people have started to work on a feature, it should be marked [In Progress](https://github.com/metabase/metabase/labels/In%20Progress). Once there is a branch+some code, a pull request is opened, linked to the feature + any issues that were pulled together to inform the feature.
 
 ### E) Verification and merging
 
 All PRs that involve more than an insignificant change should be reviewed. See our [Code Review Process](code-reviews.md).
- 
-If all goes well, the feature gets coded up, verified and then the pull request gets merged! High-fives all around. 
 
-If there are tests missing, code style concerns or specific architectural issues in the pull request, they should be fixed before merging. We have a very high bar on both code and product quality and it's important that this be maintained going forward, so please be patient with us here. 
+If all goes well, the feature gets coded up, verified and then the pull request gets merged! High-fives all around.
+
```

Comment:
but you like newlines? ;)

Reply:
https://github.com/metabase/metabase/blob/a946e53c1d7fb51f14535e4766cfc1467a824c43/.editorconfig#L9